osu.Game.Modes.Osu/UI/OsuHitRenderer.cs

```diff
@@ -17,7 +16,7 @@ public OsuHitRenderer(Beatmap beatmap)
         {
         }
 
-        protected override HitObjectConverter<OsuHitObject> Converter => new OsuHitObjectConverter();
+        protected override PlayMode PlayMode => PlayMode.Osu;

```

Comment:
not a fan of this. playmode enums should not be added to more places in the code base (beacuse they are going to be removed soon)